optimum/neuron/modeling_diffusion.py

```diff
@@ -541,16 +577,15 @@ def _export(
         force_download: bool = True,
         cache_dir: Optional[str] = None,
         compiler_workdir: Optional[str] = None,
-        inline_weights_to_neff: bool = True,
+        disable_neuron_cache: Optional[bool] = False,
```

Comment:
Why `Optional`? It's described as `bool` only in the docstring below.

Reply:
sort of typo, it can't be None.